### DISCUSSION TITLE
hutool有没有考虑做kotlin语言的版本

### DISCUSSION THREAD

**Discussion by surkaa** [looly/hutool#3322] [Q&A] [answered]
kotlin对java的函数式编程做了非常大的提升, 在做安卓开发时深得码农心, 请问hutool对kotlin方向有什么看法

**Comment by looly** [comment, marked as answer]
理论上Java的大部分方法和工具类在kt下都可以使用，现在Hutool对kt是有限支持，即类似JSON支持，其它单独针对kt没有考虑。

而且Hutool专注于Java工具，暂时没有考虑出kt工具。
